e2e/automated.test.js

```diff
@@ -43,14 +43,21 @@ describe('End to End tests', () => {
   });
 
   describe('single blob', () => {
-    fixtures.forEach(({ url, content, lineNumber, targetUrl }) => {
+    fixtures.forEach(({ url, file, content, lineNumber, targetUrl }) => {
       it(`resolves ${content} to ${targetUrl}`, async () => {
         if (lineNumber) {
-          await executeTest(
-            url,
-            targetUrl,
-            `#LC${lineNumber} .octolinker-link`,
-          );
+          try {
+            await executeTest(
+              url,
+              targetUrl,
+              `#LC${lineNumber} .octolinker-link`,
+            );
+          } catch (ex) {
+            process.stdout.write(
+              `\n::error file=${file},line=${lineNumber}::${ex.message}\n`,
+            );
```

Comment:
😍

Reply:
I think a `console.log` should work here too. That way you can get rid of the newline operator.